src/Wallabag/CoreBundle/Helper/EntriesExport.php

```diff
@@ -300,13 +304,38 @@ private function producePdf()
                 $pdf->SetKeywords($tag->getLabel());
             }
 
+            $publishedBy = $entry->getPublishedBy();
+            if (!empty($publishedBy)) {
+                $authors = implode(',', $publishedBy);
+            } else {
+                $authors = $this->translator->trans('export.unknown');
```

Comment:
You can avoid the `else` and define `$authors = $this->translator->trans('export.unknown');` before the `if`